### PR TITLE
[FCE-425] Rename screencast to screen share

### PR DESCRIPTION
**Pull request by maksg**
## Description

Rename all occurrences of screencast to screen share.

## Motivation and Context

Mobile terminology should match the one on web.

### CODE REVIEW

File: packages/react-native-client/src/hooks/useFishjamEvent.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { nativeModuleEventEmitter } from '../RNFishjamClientModule';
 export const ReceivableEvents = {
   IsCameraOn: 'IsCameraOn',
   IsMicrophoneOn: 'IsMicrophoneOn',
-  IsScreencastOn: 'IsScreencastOn',
+  isScreenShareOn: 'isScreenShareOn',
```
Comment (mironiasty) [inline packages/react-native-client/src/hooks/useFishjamEvent.ts:7]:
```suggestion
  IsScreenShareOn: 'IsScreenShareOn',
```

**Review by mironiasty — COMMENTED** [review]
App is building on both platforms 🎉 
It might require more testing, but looks ok for now :)
There is one blocking issue, but once fixed PR should be good to merge

**Comment by maksg** [comment]
Done!

**Review by mironiasty — APPROVED** [review]
👀 👍 2️⃣ 👦

**Review by MiloszFilimowski — APPROVED** [review]

**Pull request merged**